azure-nextgen-ts-webapp-privateendpoint-vnet-injection/index.ts

```diff
@@ -0,0 +1,158 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+// pvt DNS zone and virtual network link are only available on this version
+import * as pvtnetwork from "@pulumi/azure-nextgen/network/v20180901";
```

Comment:
Does it have to be this old version? Have you tried `latest` for all?

Reply:
Latest works.